data/hai/hai reveal 4 middle.txt

```diff
@@ -1508,27 +1508,115 @@ mission "Hai Reveal [A14] A Time of Trouble"
+			`	Teeneep explains, "I speak as the voice of the Hai and with the full authority of the Elder Council. Thousands of our people have been kidnapped over the past few decades, and they still largely reside under the servitude of various pirate factions that the Republic allows to continue. Even worse, your authorities have allowed numerous Hai to be slipped past on slaver ships, without attempting rescue because of the no boarding policy your forces employ, even on disabled and potentially doomed ships. We demand the return of our people and an immediate end to the policy of letting slaves die or escape without attempting boarding and rescue."`
+			choice
+				`	"Thank you, Hai representative. It is unfortunate that your people have suffered in that way, and I'm sure your esteemed colleagues from the other side of the wormhole can help you solve this problem. President? Chancellor?"`
+					goto formality2
+				`	"Thank you, Teeneep. President, Chancellor, please explain the piracy situation from the human point of view."`
+			`	The President seems surprised by the informal tone of your question, as if you just made a faux pas. The Chancellor looks more at ease than before, and perhaps a little amused.`
+			label formality2
 			`	The President is the first to respond. "The problem of piracy in human space is a difficult one and cannot be solved overnight. The pirates are well-entrenched in their planets, and fighting them is a long and brutal struggle. With our navies distracted fighting one another, our recent war has left them stronger than ever."`
-			`	The Chancellor continues, "The Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`
-			`	"We have a counter-proposal," the President says. "We cannot afford to change that policy in totality, it would cost too many lives. However, we propose the following: The Free Worlds and Republic will establish a customs service on Mountaintop. Hai visiting human space will check in with us there. Our intelligence agencies will then track their movements in order to see to their safety as best we can. They'll also be given contacts on every human planet that they can go to for help. In the meantime, we will place extensive surveillance fleets at Ultima Thule to detect any unapproved Hai travel. We will do our best to disable and capture ships with kidnapped Hai, no matter the risk to Navy lives."`
+			`	The Chancellor states, "However, the Free Worlds has made great strides reducing piracy by befriending the peoples of the anarchist worlds. We are discussing similar approaches with the Republic. Perhaps in another few years, piracy will be entirely eliminated."`
+			choice
+				`	"With respect, Chancellor, what you propose would take years to resolve a problem of immediate concern. How does the Hai representative respond?"`
```

Comment:
The chancellor just said it would take years.
```suggestion
				`	"With respect, Chancellor, this is a problem of immediate concern. I'm sure the Hai representative would like to respond to that before we continue."`
```